code/game/turfs/flooring/flooring_premade.dm

```diff
@@ -419,7 +425,7 @@
 	movement_cost = 0
 
 #define FOOTSTEP_SPRITE_AMT 2
-
+/* YW Edit: Disabled until we fix the dang mobs triggering this. I'm lookin' at you kel.
```

Comment:
oof ouch owie

Reply:
You didn't fix it! Mobs still cause it, and all the snow is just footprints :c